tests/torch/data/search_building_block/SWIN MS

```diff
@@ -0,0 +1,641 @@
+[
```

Comment:
I wanted to comment on that earlier, but didn't get the chance to.

There's a lot of files in NAS-related code that are named entirely wrong. The file names should be lower-case only to avoid potential issues on Windows, should not contain spaces and should ideally have an extension. This seems like a JSON format, so a `.json` extension would be good.

Reply:
Corrected file names